src/components/input/SelectNext.tsx

```diff
@@ -49,20 +64,27 @@ function SelectOption<T>({
   return (
     <li
       className={classnames(
-        'w-full ring-inset focus:ring outline-none rounded-none cursor-pointer',
+        'w-full ring-inset outline-none rounded-none',
         'border-t first:border-t-0 transition-colors whitespace-nowrap',
-        { 'hover:bg-grey-1': !disabled },
+        {
+          'text-grey-4': disabled,
```

Comment:
I have used `text-grey-4` instead of `text-grey-5` as suggested, as this is the color we use for other components when disabled.